controllers/getEndpoints.js

```diff
@@ -0,0 +1,12 @@
+const { fetchEndpoints } = require('../models/fetchEndpoints')
+
+function getEndpoints(req, response) {
+
+    fetchEndpoints().then((endpoints) => {
+        response.status(200).send(endpoints);
```

Comment:
you should send back endpoints on a key. something like ".send({data: endpoints})"